### DISCUSSION TITLE
Async does not work with parameters

### DISCUSSION THREAD

**Discussion by craigg75** [Akryum/v-tooltip#393] [Q&A] [answered]
This works:
`<div v-tooltip="{content:asyncMethod}></div>`

This does not work:
`<div v-tooltip="{content:asyncMethod()}></div>`
nor does this:
`<div v-tooltip="{content:asyncMethod(99)}></div>`

**Comment by Ruinevo** [comment, marked as answer]
it works for me:
`<div v-tooltip="{content: asyncMethod.bind(null, 99) }></div`

**Reply by Akryum** [reply]
To add to this answer, the `content` property expects a function, so it can call it. You shouldn't call it yourself in the Vue template.
